### ISSUE TITLE
Docs - Policies Schema

### ISSUE THREAD

**Issue by jomalsan** [cloud-custodian/cloud-custodian#4605]
Looking for a top level documentation page that describes all of the available inputs to a policies yaml file. From looking through examples I have found "name", "description", "modes", "filters" and "actions", but do not know if this is a complete list

**Comment by kapilt** [comment]
at the moment its

`vars` and `policies` as top level keys in the file. for policies, its 

 - name
 - title
 - description
 - tags
 - mode
 - filters
 - resource
 - actions

tags is used for policy organizational purposes and filtering.

given this information is generic, destination for this should be one of the top level / provider independent tutorials/glossaries.

**Comment by stefangordon** [comment]
Yeah mode and specifically `execution-mode` I have found to be one that people really struggle to track down, and tbh it is one of the most critical things.     We kind of hide some of that inside Lambda and Functions specific docs.

**Comment by kapilt** [comment]
an anatomy bit would be useful, another useful bit would be having better reference doc rendering around modes, at the moment we capture schema but not doc strings. it would be nice if we could render them akin to what we do for filters/actions/resources.

also my enumeration also left of a couple of bits like max-resources and start/stop/region (conditions).
